test-apps/imodel-transformer/src/Main.ts

```diff
@@ -68,6 +70,7 @@ interface CommandLineArgs {
     // print/debug options
     Yargs.option("logChangeSets", { desc: "If true, log the list of changeSets", type: "boolean", default: false });
     Yargs.option("logNamedVersions", { desc: "If true, log the list of named versions", type: "boolean", default: false });
+    Yargs.option("extraValidation", { desc: "If true, perform extra validation to confirm there were no underlying issues with the source and no transformation issues with the target", type: "boolean", default: false });

```

Comment:
`extraValidation` sounds like it should either have a more specific name to make it understandable why it isn't part of normal validation, or it should just be plain `validation`/`validate` probably.

Reply:
I will rename to `validation` and update the description to make it clear that the option is only for reproducing issues or confirming results (extra, not required).